opendut-carl/opendut-carl-api/Cargo.toml

```diff
@@ -8,18 +8,12 @@ license.workspace = true
 [features]
 default = ["codegen"]
 codegen = ["tonic/prost", "tonic/codegen", "tonic-build/prost"]
-oidc_client = [
-    "dep:anyhow",
-    "dep:config",
-    "dep:oauth2",
-    "dep:shadow-rs",
-]
 client = [
     "codegen",
     "dep:chrono",
     "dep:config",
     "dep:http",
-    "dep:oauth2",
+    "opendut-auth/confidential_client",
```

Comment:
See above. This is a change from another commit that does not belong into this PR. To be removed.